source/features/mark-merge-commits-in-list.tsx

```diff
@@ -34,7 +34,7 @@ const filterMergeCommits = async (commits: string[]): Promise<string[]> => {
 
 // eslint-disable-next-line import/prefer-default-export
 export function getCommitHash(commit: HTMLElement): string {
-	return commit.querySelector('a[href]')!.pathname.split('/').pop()!;
+	return commit.querySelector('a[href]:not(.js-issue-link)')!.pathname.split('/').pop()!;
```

Comment:
If there's an easier element to select, let's not go by exclusion with `not`

```suggestion
	return select('[aria-label="Copy the full SHA"]', commit)!.getAttribute('value');
```